website/client/components/achievements/emptyArmoryEnabled.vue

```diff
@@ -0,0 +1,33 @@
+<template lang="pug">
+  b-modal#empty-armory-enabled(:title="$t('emptyArmoryNew')", size='md', :hide-footer="true")
+    .modal-body
+      .col-12
+        .armory_key
+        p
+          span {{ $t('emptyArmoryUnlock') }}
```

Comment:
This same modal seems to appear when the user reaches level 50 and when they reach level 100. Should the message perhaps be a little different in the latter case, so it's clear the feature is now free to use?

Reply:
No the modal doesn't reappear in level 100 (the rebirth doesn't too, so I just applied the same here). Should that change?